lib/reporters/helpers/enhanced-assertion.js

```diff
@@ -0,0 +1,20 @@
+var chalk = require('chalk');
+
+function EnhancedAssertion() {
+	if (!(this instanceof EnhancedAssertion)) {
+		return new EnhancedAssertion();
```

Comment:
This should throw: `new EnhancedAssertion()` should be used instead. See [`Hook`](https://github.com/avajs/ava/blob/4d3ed27865dc7cdfde7e651711ee4cd0646ea6e8/lib/hook.js#L8) for an example.